test/bftest.c

```diff
@@ -434,19 +434,35 @@ static int test_bf_ofb64(void)
 }
 #endif
 
+typedef enum OPTION_choice {
+    OPT_ERR = -1,
+    OPT_EOF = 0,
+    OPT_PRINT = 1
+} OPTION_CHOICE;
+
+static const OPTIONS test_options[] = {
+    { "print", OPT_PRINT, '-', "Output test tables instead of running tests"},
+    { NULL }
+};
+const OPTIONS *test_get_options(void)
+{
+    return test_options;
+}
```

Comment:
Style wise, this deviates from the others I see, where you have the definition of `test_options` inside `test_get_options`. Perhaps pick one style and stick to it?

Note: just a comment, not a request. It's your submission and neither variant goes against our coding style, so...

Reply:
Will change..